community/OM-Operator-Management/cnv-samples/cnv-operator-install/policy-cnv.yaml

```diff
@@ -0,0 +1,179 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: openshift-virtualization
+  namespace: default
+spec:
+  remediationAction: enforce
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: kubevirt-namespace
```

Comment:
In the current version of ACM, OperatorPolicy will automatically create this namespace if necessary. If no special labels or other configuration are needed on the namespace, I would prefer to omit the namespace from the policy. If you need to support a specific older version of ACM, I can check on when exactly we added this behavior.

Reply:
Will do.